sdk/compute/azure-mgmt-compute/generated_samples/virtual_machine_scale_set_examples/virtual_machine_scale_set_create_with_priority_mix_policy.py

```diff
@@ -36,12 +36,14 @@ def main():
             "location": "westus",
             "properties": {
                 "orchestrationMode": "Flexible",
+                "platformFaultDomainCount": 1,
```

Comment:
`generated_samples` are generated with swagger examples so we shall fix it in swagger: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_Create_WithPriorityMixPolicy.json